scaleway/data_source_baremetal_offer_test.go

```diff
@@ -64,6 +64,124 @@ func TestAccScalewayDataSourceBaremetalOffer_Basic(t *testing.T) {
 	})
 }
 
+func TestAccScalewayDataSourceBaremetalOffer_SubscriptionPeriodHourly(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					data "scaleway_baremetal_offer" "test1" {
+						zone = "fr-par-2"
+						name = "EM-A210R-HDD"
+
+						subscription_period = "hourly"
+					}
+					
+					data "scaleway_baremetal_offer" "test2" {
+						zone     = "fr-par-2"
+						offer_id = "25dcf38b-c90c-4b18-97a2-6956e9d1e113"
```

Comment:
Could you have this offer id coming out of a data resource? It makes it easier to maintain in case the id change on the elastic metal team